demo/src/screens/componentScreens/CardsScreen.js

```diff
@@ -43,43 +52,47 @@ export default class CardsScreen extends Component {
               })}
             </ScrollView>
 
-            <Card height={120} width={120} marginV-20>
-              <Card.Image height={'100%'} imageSource={cardImage} />
-            </Card>
+            <View row>
+              <Card height={120} flex marginV-20 marginR-20>
```

Comment:
I would give this one `selected` as well since they are in the same row...

Reply:
done